src/ansi-c/expr2c.cpp

```diff
@@ -2173,22 +2239,24 @@ std::string expr2ct::convert_array(
 
   // we treat arrays of characters as string constants,
   // and arrays of wchar_t as wide strings
-
-  const typet &subtype=ns.follow(ns.follow(src.type()).subtype());
```

Comment:
It's a bit hard to tell in the review, but I think ns.follow is again unnecessary here (the inner one definitively is)